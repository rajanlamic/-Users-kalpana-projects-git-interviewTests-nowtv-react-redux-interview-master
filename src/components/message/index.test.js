import React from 'react';
import { shallow } from 'enzyme';
import Message from '.';

describe('message', () => {
  let wrapper, props;

  beforeEach(() => {
    props = {
      message: 'test message',
      avatar: 'http://dummyimage.com/100x100.png/5fa2dd/fffff',
      timestamp: '2016-02-24T00:01:28Z',
      email: 'test@email.com',
    };
    wrapper = shallow(<Message {...props} />);
  });

  afterEach(() => {
    props = null;
    wrapper = null;
  });

  it('should render Message property', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
