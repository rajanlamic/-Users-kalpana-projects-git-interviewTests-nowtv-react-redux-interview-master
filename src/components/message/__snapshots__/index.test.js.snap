// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`message should render Message property 1`] = `
<li
  className="message"
>
  <div
    title="test@email.com"
  >
    test message
  </div>
  <div>
    <img
      alt="http://dummyimage.com/100x100.png/5fa2dd/fffff"
      src="http://dummyimage.com/100x100.png/5fa2dd/fffff"
    />
  </div>
  <div>
    24-02-2016 00:01
  </div>
</li>
`;
